src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -182,7 +182,7 @@
     },
     "constants": {
       "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestServerProjects\\constants\\Generated"
+      "commandLineArgs": "--standalone $(SolutionDir)\\\\test\\\\CadlRanchProjects\\\\arrays\\\\item-types\\\\Generated"
```

Comment:
Why we change this?

Reply:
This is unexpected I will revert this. This is for local debug